doc/integration.md

```diff
@@ -103,10 +104,87 @@ Building upon the code sample from the previous section, consider the following
+```
+
+Assume `storedToken` contains the user's token as received during registration. Note that the method expects the purchase amount to be supplied in cents. You can find the method's full documentation [here](http://cocoadocs.org/docsets/Klarna-on-Demand/0.1.2/Classes/KODOriginProof.html#//api/name/generateWithAmount:currency:userToken:).
+
+###Purchase example
+We now know how to generate the signature required for a purchase to go through. Let us see how to send it, along with other required information, to the sample backend.
+
+You will most likely have a "buy" button somewhere in your application. The code below shows how such a button might be implemented in your application's controller:
+
+```objective-c
+- (IBAction)onBuyPressed:(id)sender {
+  // create an origin proof, as seen in the previous section
+  NSString *originProof = [KODOriginProof generateWithAmount:9900 currency:@"SEK" userToken:storedToken];
+
+  // send the purchase request to the backend
+  AFHTTPRequestOperationManager *manager = [AFHTTPRequestOperationManager manager];
+  manager.requestSerializer = [AFJSONRequestSerializer serializer];
+
+  NSDictionary *params = @{
+    @"origin_proof" : originProof,
+    @"reference" :    @"TCKT0001",
+    @"user_token" :   storedToken
+  };
+
+  [manager POST:@"http://localhost:9292/pay" parameters:params
+        success:^(AFHTTPRequestOperation *operation, id responseObject) {
+          // The purchase was successful!
+        }
+        failure:^(AFHTTPRequestOperation *operation, NSError *error) {
+          // Something bad happened
+        }
+  ];
+}
+```
+
+The code above, which utilizes the [AFNetworking](http://afnetworking.com/) framework, is less daunting than it seems. All it does is send the following JSON to `http://localhost:9292/pay` (where the sample backend expects purchase requests when run locally):
+
+```json
+{
+  "origin-proof":"eyJkYXRhIjoie1wiYW1vdW50XCI6OTkwMCxcImN1cnJlbmN5",
+  "reference":"TCKT0001",
+  "user_token":"c4efa3a2-3c02-4544-9259-720285788f60"
+}
+```
+
+Remember that if you try this out for yourself, your origin proof and user token will obviously be different. Also note the placeholder comments in the "success" and "failure" blocks above, where you will most likely want to notify the user of the purchase attempt's outcome.
+
+This is really all there is to performing a purchase, though as previously mentioned you will want to take a look at the [sample backend](https://github.com/klarna/sample-app-backend) to get the full picture.
+
 ##The preferences view
 After having registered to pay using Klarna, users may wish to view or even alter their payment settings (for example, users may wish to switch from using a credit card to monthly invoice payments). As was the case with registration, the SDK provides a view for this purpose. Using the user token acquired during the registration process, you will be able to present your users with a preferences view.
 
```

Comment:
It's important to mention that changing the preferences does not alter the user token.